src/main/msp/msp.h

```diff
@@ -35,6 +44,7 @@ typedef enum {
 typedef struct mspPacket_s {
     sbuf_t buf;
     int16_t cmd;
+    uint8_t flags;
```

Comment:
`flags` and `result` are a bit complementary in V2 ...

Reply:
Why? `flags` is a placeholder for MSPv2 flags field (unused at the moment, but can be used to i.e. fetch defaults instead of current values).

Reply:
One bit (error flag) is transferred in preamble, other flags in flag field. (And there is no standard way to report error code)

Reply:
BTW: fetch default is related to command, not protocol. So command shall define related flags for this use

Reply:
That's the idea. Flags are extension for command, not protocol